### PR TITLE
New Rviz trajectory playback display

### PR DESCRIPTION
**Pull request by davetcoleman**
This is the combined RR for #478 that removes the trajectory playback functionality from the `MotionPlanningRvizPlugin` into a reusable `TrajectoryVisualization` class that can be used both inside the `MotionPlanningRvizPlugin`exactly as it was before, and in a new `TrajectoryRvizPlugin` that isolates the functionality similar to how we already have for RobotState and PlanningScene.

It has the exact same features as the original 'Planned Path' property panel, except I fixed a bug with the alpha property not being used correctly at startup.

In other words, the same code is used now in two different displays:
screenshot from 2015-01-09 00 07 56

### CODE REVIEW

**Comment by mikeferguson** [comment]
@davetcoleman I'm out of the office until monday afternoon -- but I'll give this a runtime test then.

**Comment by davetcoleman** [comment]
Ping - I'd like to see this work merged in

**Comment by mikeferguson** [comment]
building it right now.....

**Comment by mikeferguson** [comment]
Ok, works well for me.
